bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -1083,7 +946,7 @@ spec:
         resources:
           requests:
             cpu: 10m
-            memory: 300Mi
+            memory: 600Mi
```

Comment:
this is measured and justified? or its just combination of 300+300 of both containers?